utils/score_der.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2021 Johns Hopkins University (Jiatong Shi)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+collar=0.0
+frame_shift=128
+fs=8000
+subsampling=1
+
+./utils/parse_options.sh || exit 1
+
+if [ $# -lt 3 ]; then
+    echo "Usage: $0 <scoring_dir> <infer_scp> <gt_label>";
+    exit 1;
+fi
+
+scoring_dir=$1  # scoring directory
+infer_scp=$2    # inference scp from diar_inference
+gt_label=$3     # ground truth rttm
+
+echo "Scoring at ${scoring_dir}"
+mkdir -p $scoring_dir || exit 1;
+
+for med in 1 11; do
+    for th in 0.3 0.4 0.5 0.6 0.7; do
+        make_rttm.py --median=$med --threshold=$th \
+            --frame_shift=${frame_shift} --subsampling=${subsampling} --sampling_rate=${fs} \
```

Comment:
Sorry, I missed. 
We don't use `utils` way any more. This is one of the bad point of espnet1 because the script doesn't show the real path of the script, it makes hard to find script paths.

Please use `scripts` and `pyscripts` under the template directory instead.